x-pack/plugins/endpoint/public/embeddables/resolver/models/process_event_test_helpers.ts

```diff
@@ -4,33 +4,46 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { ProcessEvent } from '../types';
-type DeepPartial<T> = { [K in keyof T]?: DeepPartial<T[K]> };
+import { LegacyEndpointEvent } from '../../../../common/types';
 
+type DeepPartial<T> = { [K in keyof T]?: DeepPartial<T[K]> };
 /**
  * Creates a mock process event given the 'parts' argument, which can
  * include all or some process event fields as determined by the ProcessEvent type.
  * The only field that must be provided is the event's 'node_id' field.
  * The other fields are populated by the function unless provided in 'parts'
  */
-export function mockProcessEvent(
-  parts: {
-    data_buffer: { node_id: ProcessEvent['data_buffer']['node_id'] };
-  } & DeepPartial<ProcessEvent>
-): ProcessEvent {
-  const { data_buffer: dataBuffer } = parts;
+export function mockProcessEvent(parts: {
+  endgame: {
+    unique_pid: LegacyEndpointEvent['endgame']['unique_pid'];
+    unique_ppid?: LegacyEndpointEvent['endgame']['unique_ppid'];
+    process_name?: LegacyEndpointEvent['endgame']['process_name'];
+    event_subtype_full?: LegacyEndpointEvent['endgame']['event_subtype_full'];
+    event_type_full?: LegacyEndpointEvent['endgame']['event_type_full'];
+  } & DeepPartial<LegacyEndpointEvent>;
+}): LegacyEndpointEvent {
+  const { endgame: dataBuffer } = parts;
   return {
-    event_timestamp: 1,
-    event_type: 1,
-    machine_id: '',
-    ...parts,
-    data_buffer: {
-      timestamp_utc: '2019-09-24 01:47:47Z',
+    endgame: {
```

Comment:
reminder: we need tests that render when events are in weird order. and we need tests that say that events are rendered in a stable way based on timestamps.